### ISSUE TITLE
data 

### ISSUE THREAD

**Issue by xwybot** [nizhf/hoi-prediction-gaze-transformer#6]
Is the training set and validation set somewhere/VidOR/video/? What about the numbering conflict？

**Comment by nizhf** [comment]
Hi. Thank you for your interest in our work. 

All videos for training and validation are in somewhere/VidOR/video. The original annotations from VidOR for these videos are in somewhere/VidOR/annotation. In the annotations, you can see the training and validation set splits. 

And what do you mean about the numbering conflict?
